lib/awful/spawn.lua

```diff
@@ -370,6 +370,18 @@ function spawn.easy_async(cmd, callback)
     })
 end
 
+--- Call spawn.easy_async with a shell.
+-- This calls `cmd` with `$SHELL -c` (via `awful.util.shell`).
+-- @tparam string cmd The command.
+-- @tab callback Callback function.
```

Comment:
See https://github.com/awesomeWM/awesome/pull/1001 for improving easy_async by itself first.